power_specs.py

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python
 # vim: set fileencoding=UTF-8 :
-
+#pylint: disable= R, C
+#pylint: disable=no-member
+# pylint: disable=W0622
+# pylint: disable=W0105
```

Comment:
Please solve this one too, which is about moving the docstrings below the function definition.